config.py

```diff
@@ -1,34 +1,46 @@
 import argparse
 import configparser
 import logging
+from logging import debug, info, error, warning, critical
 import os
+import sys
 
 
 def parse_arguments(args):
     parser = argparse.ArgumentParser()
     parser.add_argument('--log-level', metavar="LEVEL", type=str, help="Log level", default=None)
+    parser.add_argument('--config', metavar="FILE", type=str, help='Config file.')
     parser.add_argument('rest_args', metavar="ARGS", nargs='*')
     ns = parser.parse_args(args)
     return vars(ns)
 
 
-def parse_configfile():
+def parse_configfile(config_file):
     cfg = configparser.ConfigParser()
-    cfg.read(os.path.expanduser("~/.rtorrent_low_space_driver.cf"))
+    with open(config_file, 'r') as f:
+        cfg.read_file(f)
     return cfg
 
 
 def start_logger(ns, cfg):
     log_level = ns.get('log_level') or cfg.get('main', 'log_level', fallback='INFO')
     logging.basicConfig(
         level=getattr(logging, log_level),
-        format="%(asctime)s - %(levelname)8s - %(name)s - %(message)s"
+        format="%(asctime)s - %(levelname)8s - %(name)s - %(message)s",
+        force=True
     )
```

Comment:
force=True is necessary because of pytest. It is possible to keep support for python 3.6+ if we use an advanced logger, I did it before, the code isn't much more complicated.

Reply:
Yes I can certainly do that, but before I commit again, let me ask your input on this: 
- `force=True` is the sole reason we lose compatibility with 3.6=>, if we use "advanced logger" we retain this compatibility, the code will become a bit longer but we retain compatibility, and I have it mostly ready in my pc. Python compatibility is not a problem for me since I use Archlinux, but thinking of other users, Ubuntu's Bionic ships with 3.6 and I just read that the latest Raspbian ships 3.7. 
- Honestly if you ask me, I'd implement advanced logging, pushing one more commit, because I feel the compatibility thing is a big deal. From your point-of-view, how do you feel?

Reply:
Thanks Gustavo.  I think I misunderstood your original message: I thought by "advanced logger" you meant a different logging library.  But reading back, I think you mean another method of configuring the standard `logging` module that just doesn't use the `basicConfig` method.  The latter is definitely preferable: I'm certainly not attached to the `basicConfig` method, it was just the easiest method at the time, but any other logging setup technique is fine as well.  I agree with your points regarding backwards compatibility.  :+1: